sources/constraint-declaration-validation.asciidoc

```diff
@@ -893,17 +893,19 @@ Examples for types being subject to implicit unwrapping are `java.util.OptionalI
 @Min(1)
 private OptionalInt optionalNumber;
 
-@Size(max=10)
-private StringProperty name;
+@Negative
+private LongProperty negativeLong;
 
 @Positive
 private IntegerProperty positiveInt;
+
+private final ListProperty<@NotBlank StringProperty> notBlankStrings;
 ----
 ====
 
-Here the `@Min`, `@Size` and `@Positive` constraints don't apply to the annotated `OptionalInt`, `StringProperty` and `IntegerProperty` themselves, but rather to the wrapped string and integer values, respectively.
+[tck-testable]#Here the `@Min`, `@Negative`, `@Positive` and `@NotBlank` constraints don't apply to the annotated `OptionalInt`, `LongProperty`, `IntegerProperty` and `StringProperty` objects themselves, but rather to the wrapped numeric and string values, respectively.#
```

Comment:
Ok, no problem. Feel free to remove the assertion marker before merging then. I did it mainly for the list of inter property example. I've sent a PR towards your TCK repo for adding this.